model.py

```diff
@@ -0,0 +1,64 @@
+from sqlalchemy import Column, ForeignKey, Integer, String, DateTime
+from sqlalchemy.orm import relationship
+from sqlalchemy.ext.declarative import declarative_base
+
+# Create the base class for declarative models
+Base = declarative_base()
+"""
+Base class provides a set of common functionality for declarative models.
+It is used as a base class for our User and Upload classes to inherit from.
+This allows SQLAlchemy to track and manage the defined models.
+"""
+
+
+class User(Base):
+    """
+    Class User represents a user in the system with a unique email address.
+    It has a one-to-many relationship with the Upload class, representing the files uploaded by the user.
+    """
+
+    # Define the table name for the User model
+    __tablename__ = "users"
+
+    # Primary key column for the User model
+    id = Column(Integer, primary_key=True)
+
+    # Email column with unique constraint and non-nullability
+    email = Column(String, unique=True, nullable=False)
+
+    # Relationship with the Upload model, specifying the back reference and cascade behavior
+    uploads = relationship("Upload", back_populates="user", cascade="all, delete")
+
+
+class Upload(Base):
```

Comment:
No need to over-do it with the comments. most columns explain themselves and need no documentation